test/adapter/json/polymorphic_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    module Adapter
+      class Json
+        class PolymorphicTest < ActiveSupport::TestCase
+          def setup
```

Comment:
@bf4 Agreed, I also favor it. I just wanted to make it clear to @marcgarreau that it was an improvement suggestion rather than error correction.